examples/fragment-performance/test.js

```diff
@@ -0,0 +1,86 @@
+const puppeteer = require('puppeteer');
+const assert = require('assert');
+const http = require('http');
+
+const Tailor = require('../../index');
+const serveFragment = require('../multiple-fragments-with-custom-amd/fragment');
+const { analyseHooks } = require('./hooks');
```

Comment:
since the function inside hooks is IIFE, its getting executed twice in this test.https://github.com/zalando/tailor/blob/654b4b83b4809775d8bc5962206d0595c8a417b5/examples/fragment-performance/hooks.js#L4

Can you extract that function and use the same in test as well. 



Check travis logs - https://travis-ci.org/zalando/tailor/jobs/548757347